Formula/g/gnu-getopt.rb

```diff
@@ -4,6 +4,7 @@ class GnuGetopt < Formula
   url "https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.39/util-linux-2.39.2.tar.xz"
   sha256 "87abdfaa8e490f8be6dde976f7c80b9b5ff9f301e1b67e3899e1f05a59a1531f"
   license "GPL-2.0-or-later"
+  revision 1
```

Comment:
No need to force a reinstall for all users for this

Reply:
Should I take it out? How will people get the new files?

Reply:
I think the argument might be it is not significant enough to warrant an revision bump. It wold be picked up for the next version. (also, so far no one has complained yet? do you know when did it get introduced?)

Reply:
They can get the new file but reinstalling or waiting for the next release

Reply:
@chenrui333 The examples have been part of util-linux since at least 2006. Since this formula is manually extracting and installing `getopt` from util-linux, this was most likely an oversight.

@SMillerDev Ok, I'll take the revision out and push.

Reply:
Oh, it's already been removed.